h2o-py/h2o/transforms/decomposition.py

```diff
@@ -123,20 +122,20 @@ def __init__(self, nv=None, max_iterations=None, transform="NONE", seed=None,
             level of every categorical variable will be dropped. Defaults to True.
         :param str svd_method: A character string that indicates how SVD should be calculated. Possible values are:
 
-            - ``"GramSVD"``: distributed computation of the Gram matrix followed by a local SVD
+            - ``"gram_svd"``: distributed computation of the Gram matrix followed by a local SVD
               using the JAMA package,
-            - ``"Power"``: computation of the SVD using the power iteration method,
-            - ``"Randomized"``: approximate SVD by projecting onto a random subspace.
+            - ``"power"``: computation of the SVD using the power iteration method,
+            - ``"randomized"``: approximate SVD by projecting onto a random subspace.
 
         :returns: a new H2OSVD model
         """
         super(H2OSVD, self).__init__()
         self._parms = locals()
         self._parms = {k: v for k, v in self._parms.items() if k != "self"}
 
-        assert_is_type(svd_method, Enum("GramSVD", "Power", "GLRM", "Randomized"))
+        assert_is_type(svd_method, None, Enum("gram_s_v_d", "power", "randomized"))
```

Comment:
`gram_s_v_d` looks like a mistake.

Reply:
I'm just using the exact same naming as in `estimators/svd.py`: not planning to fix every single nitpick from legacy code here.
Note that `_` are ignored both in type check and on backend side when resolving values to Java enum.

Reply:
PCA  supports `GRLM` but SVD apparently doesn't: either it was a mistake (copy/paste from PCA) in this file, or the SVD estimator is missing it.

Reply:
in SVDModel:
```java
    public enum Method {
      GramSVD, Power, Randomized
    }
```